project-keeper/src/main/resources/non_maven_templates/.github/workflows/project-keeper.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly pk_mode="${1-verify}";
+readonly version="##VERSION##"
+
+readonly pk_jar="$HOME/.m2/repository/com/exasol/project-keeper-cli/$version/project-keeper-cli-$version.jar"
+
+if [ ! -f "$pk_jar" ]; then
+    echo "Downloading Using Project Keeper $version"
+    mvn --batch-mode org.apache.maven.plugins:maven-dependency-plugin:3.3.0:get -Dartifact=com.exasol:project-keeper-cli:$version
```

Comment:
maybe on interactive use you want to run mvn without `--batch-mode`?
should we move cmd line arg `--batch-mode` to the yml workflow file?